.ci/openvino-onnx/Dockerfile

```diff
@@ -40,11 +40,16 @@ RUN apt-get update && apt-get -y --no-install-recommends install \
         python3-dev \
         pybind11-dev \
         python3-virtualenv \
-        cython3 \
+        python3-venv \
         tox && \
     apt-get clean autoclean && \
     apt-get autoremove -y
 
+RUN python3 -m venv ~/.venv
+RUN . ~/.venv/bin/activate
+RUN which python3
+RUN python3 -m pip install "cython>=3.0.0" --break-system-packages
```

Comment:
looks like `. ~/.venv/bin/activate` does not work as you expected, otherwise `--break-system-packages` is not required.

Reply:
I'll create a cleanup PR today.